3_visualize/src/plot_timeseries.R

```diff
@@ -1,8 +1,9 @@
 plot_nwis_timeseries <- function(fileout, site_data_styled, width = 12, height = 7, units = 'in'){
   
+  site_data_styled <- read_feather(site_data_styled)
```

Comment:
Nice work catching that the switch of target type means that `site_data_styled` now has to be read in from the feather file. Sometimes in this scenario we'd rename the incoming argument `site_data_styled_file`, so that it's clear that the styled site data will need to be read in.

Reply:
Nice idea! I went ahead and did that; thanks.